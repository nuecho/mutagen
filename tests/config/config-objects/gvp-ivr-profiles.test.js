cfgObjectTests("gvpIVRProfiles", "gvp-ivr-profiles-config.json");