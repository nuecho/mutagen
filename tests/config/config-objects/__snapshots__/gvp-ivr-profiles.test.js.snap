// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[mutagen gvpIVRProfiles config import-export] should create the gvpIVRProfiles: the gvpIVRProfiles have been created 1`] = `
Object {
  "stderr": "",
  "stdout": "+ gvpivrprofile => cfgObjectTest
Completed. 1 object(s) imported.
",
}
`;

exports[`[mutagen gvpIVRProfiles config import-export] should throw exception when new gvpIVRProfiles to import miss mandatory properties gvpIVRProfiles: exception has been thrown 1`] = `
Object {
  "stderr": "Cannot import configuration: some configuration objects' mandatory properties for creation are not set.
  Properties [displayName, tenant] not set in gvpivrprofile [emptyCfgObjectTest].
",
  "stdout": "",
}
`;

exports[`[mutagen gvpIVRProfiles config import-export] should update the modified gvpIVRProfiles: the modified gvpIVRProfiles have been updated 1`] = `
Object {
  "stderr": "",
  "stdout": "~ gvpivrprofile => cfgObjectTest
Completed. 1 object(s) imported.
",
}
`;

exports[`[mutagen gvpIVRProfiles config import-export] should update the same gvpIVRProfiles: the gvpIVRProfiles have been updated 1`] = `
Object {
  "stderr": "",
  "stdout": "~ gvpivrprofile => cfgObjectTest
Completed. 1 object(s) imported.
",
}
`;
